mrmustard/utils/graphics.py

```diff
@@ -12,17 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""A module containing utility classes and functions for graphical display."""
+
+from copy import copy
+from typing import Tuple
 from rich.progress import Progress, TextColumn, BarColumn, TimeRemainingColumn
 import matplotlib.pyplot as plt
-from matplotlib.colors import Normalize
 from matplotlib import cm
 import numpy as np
-from mrmustard.types import *
 from mrmustard import settings
-from numba import njit
-from copy import copy
-
 
+# pylint: disable=disallowed-name
```

Comment:
What name is disallowed? 👀

Reply:
`bar`, because is used too often in examples together with `foo` and `baz`

Reply:
Ah, maybe better rename it then to e.g., `pbar` or `progress_bar`?

Reply:
No need I think, the name makes sense in the context. Also, the warning is only disabled for the `Progressbar` class.